exporter/otlpexporter/otlp_test.go

```diff
@@ -20,12 +20,12 @@ import (
 	"path/filepath"
 	"runtime"
 	"sync"
+	"sync/atomic"
```

Comment:
Shouldn't this change the exporter's go.mod file as well to remove the direct dependency?

Reply:
Done